src/test/datascience/notebook/notebookEditor.vscode.test.ts

```diff
@@ -123,69 +110,4 @@ suite('Notebook Editor tests', () => {
         // The third cell should have a runState of Success
         assert.strictEqual(thirdCell?.metadata.runState, NotebookCellRunState.Success);
     });
-
-    test('Switch kernels', async function () {
```

Comment:
Not required at all, we have tests for Kernel Selection in a separate file.
(the other tests we test switching between kernels/interpreters).